apps/meteor/client/components/BurgerMenu/BurgerMenuButton.tsx

```diff
@@ -1,35 +1,21 @@
-import { css } from '@rocket.chat/css-in-js';
-import { Box } from '@rocket.chat/fuselage';
+import { Box, IconButton } from '@rocket.chat/fuselage';
 import { useTranslation } from '@rocket.chat/ui-contexts';
 import type { ReactElement } from 'react';
 import React from 'react';
 
 import BurgerBadge from './BurgerBadge';
-import BurgerIcon from './BurgerIcon';
 
 type BurgerMenuButtonProps = {
-	open?: boolean;
 	badge?: number | unknown;
 	onClick: () => void;
 };
 
-const BurgerMenuButton = ({ open, badge, onClick }: BurgerMenuButtonProps): ReactElement => {
+const BurgerMenuButton = ({ badge, onClick }: BurgerMenuButtonProps): ReactElement => {
 	const t = useTranslation();
 
 	return (
-		<Box
-			is='button'
-			data-qa-id='burger-menu'
-			aria-label={open ? t('Close_menu') : t('Open_menu')}
-			type='button'
-			position='relative'
-			marginInlineEnd={8}
-			className={css`
-				cursor: pointer;
-			`}
-			onClick={onClick}
-		>
-			<BurgerIcon open={open} />
+		<Box position='relative'>
+			<IconButton icon='burger-menu' small data-qa-id='burger-menu' aria-label={t('Open_sidebar')} marginInlineEnd={8} onClick={onClick} />
```

Comment:
BurguerMenu looks like a sidebar button not a generic menu anymore, maybe is better to change the name of the component... WDYT?

Reply:
it was never a generic menu, not even a menu, so doesn't make sense using this name, I will change it!